mlflow/lightgbm.py

```diff
@@ -225,6 +227,38 @@ def callback(env):
                 eval_results.append(res)
             return callback
 
+        def plot_importance(features, importance, importance_type):
```

Comment:
```suggestion
        def log_feature_importance_plot(features, importance, importance_type):
```

Just to make the name a bit clearer